android/java/org/chromium/chrome/browser/upgrade/BraveUpgradeJobIntentService.java

```diff
@@ -0,0 +1,400 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
```

Comment:
that file has been removed in master

Reply:
Oh okay. I will rebase. And update the PR

Reply:
@SergeyZhukovsky it's added again with https://github.com/brave/brave-core/commit/b808f29e8452602fcf16cd7e9da4016b78141d09